Dockerfile

```diff
@@ -53,6 +54,78 @@ COPY .release/docker/docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
 ENTRYPOINT ["docker-entrypoint.sh"]
 
 
+# # By default you'll get a single-node development server that stores everything
+# # in RAM and bootstraps itself. Don't use this configuration for production.
+CMD ["server", "-dev"]
+
+
+## UBI DOCKERFILE ##
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5 as ubi
+
+ARG BIN_NAME
+# NAME and PRODUCT_VERSION are the name of the software in releases.hashicorp.com
+# and the version to download. Example: NAME=vault PRODUCT_VERSION=1.2.3.
+ARG NAME=vault
+ARG PRODUCT_VERSION
+# TARGETARCH and TARGETOS are set automatically when --platform is provided.
+ARG TARGETOS TARGETARCH
+
+LABEL maintainer="Vault Team <vault@hashicorp.com>"
+LABEL version=${PRODUCT_VERSION}
+
+# Set ARGs as ENV so that they can be used in ENTRYPOINT/CMD
+ENV NAME=$NAME
+ENV VERSION=$VERSION
+
```

Comment:
This needs to have a license to pass the [redhat certification](https://connect.redhat.com/zones/containers/container-certification-policy-guide) scan: https://github.com/hashicorp/docker-vault/blob/e4a92e52ee3d480e4fb45f26e5a7d1c295ce0d38/ubi/Dockerfile#L16-L17

Reply:
Do you know how we should provision this? The LICENSE was grabbed from externally (in the old `docker-vault`) and I was a little hesitant to do that in CRT. Should I just `wget` it here too?

Reply:
I think you'd just need to copy LICENSE from the repo root; that's how [docker-vault/ubi/Dockerfile](https://github.com/hashicorp/docker-vault/blob/e4a92e52ee3d480e4fb45f26e5a7d1c295ce0d38/ubi/Dockerfile#L16-L17) is doing it. And they're both Mozilla Public License 2.0 so I think we're good there.

Reply:
Oh d'oh. What about EULA and `TermsOfEvaluation` too? Both are also missing from this UBI copy:

https://github.com/hashicorp/docker-vault/blob/e4a92e52ee3d480e4fb45f26e5a7d1c295ce0d38/ubi/Dockerfile#L42-L43

vs our existing Alpine image:

https://github.com/hashicorp/vault/blob/main/Dockerfile#L23

Reply:
I don't think those are necessary for redhat certification. Looks like it's just doing the same thing as the library alpine image (since the downloaded release includes those files): https://github.com/hashicorp/docker-vault/blob/e4a92e52ee3d480e4fb45f26e5a7d1c295ce0d38/0.X/Dockerfile#L42-L43